ui/selectors/selectors.js

```diff
@@ -806,3 +806,25 @@ export function getIsAdvancedGasFeeDefault(state) {
     Boolean(advancedGasFee?.maxBaseFee) && Boolean(advancedGasFee?.priorityFee)
   );
 }
+
+/**
+ *  To check if the specific accountName is included in the accountsNames array which is consisted of the current user's accounts
+ *  @returns Boolean
+ */
+
+export function accountNameExists(accounts, accountName) {
```

Comment:
It doesn't seem like this needs to be a selector since its only used in new-account.component? I think you could just add it as a method on that component?

Reply:
You could use the `getAccountsNames` selector in that component and then just use includes against it.

Reply:
Fixed in 0a99efb6150d62886139fc2f1a8f2d71ea3533bd.